packages/flutter/lib/src/material/icons.dart

```diff
@@ -38,7 +38,7 @@ class IconData {
 ///
 ///  * [Icon]
 ///  * [IconButton]
-///  * <https://design.google.com/icons/>
+///  * [design.google.com/icons](https://design.google.com/icons/)
```

Comment:
<img width="250" alt="screen shot 2016-04-12 at 4 21 28 pm" src="https://cloud.githubusercontent.com/assets/1269969/14478454/aa578d4a-00ca-11e6-9f99-22d0dadad045.png">

I think it looks cleaner, but it's probably best to have all url references either use it or not, so they're consistent.

It would be pretty easy to have dartdoc cvt angle bracket references to whatever the preferred rendering is. It would happen in or around here: https://github.com/dart-lang/dartdoc/blob/76f5b7cad5ee3a47ff41de002a37c805dce9e504/lib/src/markdown_processor.dart.